test/Common/Microsoft.Test.OData.DependencyInjection/Microsoft.Test.OData.DependencyInjection.csproj

```diff
@@ -12,12 +12,9 @@
 
   <Import Project="..\Build.props" />
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net472' or '$(TargetFramework)' == 'netcoreapp2.1' or '$(TargetFramework)' == 'netcoreapp3.1' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' or '$(TargetFramework)' == 'net8.0'">

```

Comment:
Why do we need this condition yet net48 and net8.0 are the only 2 frameworks we support